server/controllers/post.controller.js

```diff
@@ -0,0 +1,80 @@
+const Post = require('../models/post');
+const cuid = require('cuid');
+const slug = require('limax');
```

Comment:
Do we need to support `cuid` and `limax`? 🤔

These packages make sense only if the user wants to build a blog, taking after our demo project right? The remaining section of users will have two libraries being installed that would not be used.

Reply:
I feel the out-of-the-box mongo `_id` can suffice. Also, forgot to mention the `sanitize-html` package as well. 🙂

Reply:
makes sense. Let's keep cuid, however. We use it in delete and get post by cuid api's 🙂